cmd/utils/manifest.go

```diff
@@ -268,17 +276,103 @@ func (*Executor) Execute(command string, env []string) error {
 	done := make(chan struct{})
```

Comment:
I think this `done` channel can be deleted. It is not being used to check when the result is added to it. Probably I left it behind when I moved this logic from the installer